src/serac/physics/solid.cpp

```diff
@@ -454,47 +463,61 @@ mfem::ParLinearForm& Solid::bulkModulusSensitivity(mfem::ParFiniteElementSpace&
 
   // Add a scalar linear form integrator using the shear sensitivity coefficient against the given bulk modulus finite
   // element space
-  if (!bulk_sensitivity_form_ || shear_sensitivity_form_->FESpace() != &bulk_space) {
-    bulk_sensitivity_form_ = std::make_unique<mfem::ParLinearForm>(&bulk_space);
+  if (!bulk_sensitivity_form_ || bulk_space) {
+    SLIC_ERROR_IF(!bulk_space, fmt::format("Finite element space is required for first sensitivity call."));
```

Comment:
Is it at all helpful to have this error mention that this is a bulk sensitivity/space?  Along with the shear one above.